### DISCUSSION TITLE
Проверка связей поля таблицы в проекте

### DISCUSSION THREAD

**Discussion by Spyker086** [totumonline/totum-ru-issues-and-discussions#348] [Как это сделать?] [answered]
Очень полезна была бы функция для конкретного поля таблицы, результатом которой была бы информация для каких сущностей проекта данное поле является источником информации.

**Comment by erpic-ru** [comment, marked as answer]
Есть такая. При попытке поменять name поля выполняется поиск связей

**Reply by ttm-alexeypolunin** [reply]
Там только надо учесть, что она не везде ищет. Например вне ее поля зрения коды форматирования таблиц и коды действия при дублировании и код-действия таблицы. Табличные в общем коды мимо.

Также она не может понять, что поле используется, когда его name определяется кодом и, скорее всего, там, где оно определяется в createRow через "", например при linkToDataTable — params — rowCreate — "some_field"

**Reply by Spyker086** [reply]
Для разбора структуры проекта было бы полезно иметь такую функцию. Я имею ввиду не через попытку замены name поля

**Reply by ttm-alexeypolunin** [reply]
Ну она просто есть уже в изменить name и выковыривать мы ее оттуда не будем :)

Чего запланировано полезного еще:

1. Сквозной поиск по кодам
2. Выполнение действия в незакрывающейся транзакцией и глушением getFromScript и notification/email send, что бы можно было посмотреть логи в боевой базе не ломая в ней данные (такой эхо тест).

Но когда это все будет — неизвестно.
